push-sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/FCMPushNotificationSender.java

```diff
@@ -26,6 +26,7 @@
 
 import javax.inject.Inject;
 
+import com.google.android.gcm.server.*;
```

Comment:
please refrain from wildcard imports. instead import `import com.google.android.gcm.server.Notification;` 

also, please organize imports so that all of the `com.google.android.gcm.server` would be placed together. 
note that there already exist an import for `import com.google.android.gcm.server.Message.Builder;` down below